templates/templates/base.html

```diff
@@ -2,6 +2,18 @@
 <!doctype html>
 <html class="no-js" lang="zh" dir="ltr">
 <head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <title>{% block title %}{{ title }}{% endblock %} | Ubuntu</title>
+
+  <link rel="stylesheet" type="text/css" media="screen" href="{% versioned_static 'css/styles.css' %}" />
+
+
+  <link rel="preconnect" href="https://www.google-analytics.com">
+  <link rel="preconnect" href="https://www.gstatic.com">
+  <link rel="preconnect" href="https://assets.ubuntu.com">
```

Comment:
This is superfluous as we preload fonts assets from the assets server nano-seconds later.